test/Concurrency/sendable_preconcurrency.swift

```diff
@@ -39,7 +39,7 @@ func testA(ns: NS, mt: MyType, mt2: MyType2, mt3: MyType3, sc: StrictClass, nsc:
   }
 }
 
-extension NonStrictStruct: @unchecked Sendable { }
+extension NonStrictStruct: @retroactive @unchecked Swift.Sendable { }
```

Comment:
Is the added `Swift.` here necessary?

Reply:
We are still permitting the previous silencing mechanism given that there won't be a way to use `@retroactive` in a way that preserves source compatibility with existing language versions, yes?

Reply:
Indeed; there's a test that covers silencing the warning via module qualification